src/content/docs/zh-cn/reference/integrations-reference.mdx

```diff
@@ -113,6 +122,19 @@ export default {
 - `clientEntrypoint` - 当组件被使用时，在客户端执行的文件的路径。这主要是为了使用 JS 渲染或填充你的组件。
 - `serverEntrypoint` - 文件路径，在服务器端请求或静态构建时，只要使用组件就会执行。这些文件应该将组件渲染成静态标记，并在适当的时候使用钩子进行激活。[React 的 `renderToString` 回调函数](https://reactjs.org/docs/react-dom-server.html#rendertostring)是个典型例子。
 
+#### `addWatchFile` 选项
+
+**类型**： `URL | string`
+
+如果你的集成依赖于一些 Vite 不监听或者需要完整的开发服务器重启才能生效的配置文件，请用`addWatchFile` 添加它。每当该文件发生变化，Astro 开发服务器将重新加载（你可以用`isRestart`检查何时发生重新加载）。
+
+使用示例：
+```js
+// 必须是绝对路径！
+addWatchFile('/home/user/.../my-config.json');
+addWatchFile(new URL('./tailwind.config.js', config.root));
+```
+
 #### `injectRoute` 选项
 
 **类型**：`({ pattern: string, entryPoint: string }) => void;`
```

Comment:
Looks like a link two lines below this changed here: https://github.com/withastro/docs/commit/db17179f3be104ac34532f7a8ef8d26ae95d6404#diff-1fee61b9ccdb2e3f9f0d612dd631d73f5d107244c69d9bf2f446a29702f29de7

Should be `/zh-cn/core-concepts/endpoints/#静态文件端点` instead of `/zh-cn/core-concepts/astro-pages/#非-html-页面`.